nextcord/application_command.py

```diff
@@ -105,9 +107,10 @@ class SlashOption:
     Parameters
     ----------
     name: :class:`str`
-        The name of the Option on Discords side. If left as None, it defaults to the parameter name.
+        The name of the Option that users will see. If left as MISSING, it defaults to the parameter name.
```

Comment:
It looks like it was always `MISSING` but the docstring appears to have been wrong

@alentoghostflame

Reply:
Pretty good chance that I overlooked this when I was initially moving things from defaulting from `None` to `MISSING`. Plz fix, thx.

Reply:
I think I'll change these docstrings to say "If not specified" instead of "If left as MISSING". The None vs MISSING stuff is not relevant to this PR and can be cleaned up later if necessary.